src/coreclr/tools/Common/TypeSystem/Ecma/SymbolReader/PortablePdbSymbolReader.cs

```diff
@@ -206,5 +207,21 @@ public override IEnumerable<ILLocalVariable> GetLocalVariableNamesForMethod(int
             }
             return variables;
         }
+
+        public override unsafe ReadOnlySpan<byte> GetSourceLinkData()
+        {
+            // {CC110556-A091-4D38-9FEC-25AB9A351A6A}
+            Guid sourceLinkDataGuid = new Guid(0xCC110556, 0xA091, 0x4D38, 0x9F, 0xEC, 0x25, 0xAB, 0x9A, 0x35, 0x1A, 0x6A);
```

Comment:
Is it better to be a static read-only field?

Reply:
RyuJIT might already optimize this to constant data. If it doesn't, it doesn't matter anyway, this is not perf critical and it's only a couple instructions to initialize.